packages/dashboard/src/components/DashboardFrame/Navigation.tsx

```diff
@@ -68,7 +87,7 @@ export function Navigation({}: NavigationProps) {
                 </ListItemIcon>
                 <ListItemText primary={t.personas()} />
             </ListItemLink>
-            <ListItem button selected={!!useRouteMatch(Routes.Wallets)} onClick={toggleNavigationExpand}>
+            <ListItem button onClick={toggleNavigationExpand}>
```

Comment:
Why? When the route path is `/wallets` the asset page is displayed and the ListItem needs to be selected

Reply:
Should we revert this?

Reply:
This PR is now in the review process and should not add new non-review changes cc @yanzhihong23

Reply:
I've revert